docs/commerce/checkout/extend_checkout.md

```diff
@@ -0,0 +1,48 @@
+---
+description: Extend checkout workflow
+edition: commerce
+---
+
+# Extend checkout workflow
+
+Depending on your needs, the [checkout](checkout.md) process can be customized, as it is highly configurable and extensible. 
+You can create [workflow](workflow.md) definitions under the `framework.workflows` [configuration key](configuration.md#configuration-files). 
+
+To start, use the default workflow that comes with the storefront module as a basis 
+and follow detailed instruction in [Customize checkout](customize_checkout.md).
```

Comment:
Do we really need to add new section? 
We already have Customize checkout section in doc (https://doc.ibexa.co/en/latest/commerce/checkout/customize_checkout) so this seems to be redundant because we are describing same things twice. 
We also have Configure checkout section (https://doc.ibexa.co/en/latest/commerce/checkout/configure_checkout/) 

Regardless of above, why are you pointing on workflow configuration for content flow since we already have described checkout workflow in: https://doc.ibexa.co/en/latest/commerce/checkout/configure_checkout/? This is completely different.